src/modules/taskLoader.js

```diff
@@ -1,14 +1,92 @@
-function TaskLoader(lists, tasks) {
-  tasks.forEach((task) => {
+/* eslint-disable no-use-before-define */
+import { stringifier } from './taskMethods.js';
+
+function TaskLoader(lists, tasks, taskEditor) {
+  // Remove tasks before re-rendering
+  while (lists.hasChildNodes()) {
+    lists.removeChild(lists.firstChild);
+  }
+  // Map tasks inside the task container
+  tasks.forEach((task, index) => {
     const list = document.createElement('li');
+    const form = document.createElement('form');
+    form.className = 'task-form';
     const checkBox = document.createElement('input');
     checkBox.type = 'checkbox';
-    const text = document.createElement('span');
-    text.innerHTML = task.string;
+    checkBox.checked = task.bool;
+    task.index = index;
+
+    // Add event listener for checkbox
+    checkBox.addEventListener('change', () => {
+      tasks[index].bool = !task.bool;
+      if (task.bool === true) {
+        text.style.textDecoration = 'line-through';
+        form.classList.remove('on');
+        dots.className = 'bi bi-three-dots-vertical';
```

Comment:
I suggest you add a delete method for an individual task. Currently this three dots don't do anything. Its part of the [requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md) having a delete method, not one that clears all completed, but one that removes individual items.
see this video

https://user-images.githubusercontent.com/48546992/199582810-ace80120-5aee-4b09-8a43-d90b5ca21319.mov

Below is the highlighted requirement:

<img width="1676" alt="Screenshot 2022-11-02 at 22 10 48" src="https://user-images.githubusercontent.com/48546992/199583569-77862c54-9bf0-48d4-8124-8ef024aa0325.png">